### PR TITLE
[PRM-2164] Added accessibility statement for penalties-appeals

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

File: conf/services/penalties-appeals [hunk 1]
```diff
@@ -0,0 +1,27 @@
+serviceName: VAT Penalties and Appeals Reform
```
Comment (LewisMitchell) [inline conf/services/penalties-appeals:1]: The IT's fail due to the file not having an extension, e.g. .yml

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/services/penalties-appeals [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+contactFrontendServiceId: vat-penalties
+complianceStatus: partial
+accessibilityProblems:
+  - On the ‘Evidence to support this appeal (optional)’ page, after users select a file to upload the markup for ‘file 1’ may mislead screen-reader users on its purpose.
+  - The ‘penalty was issued more than 30 days ago’ copy is marked up as a label and does not describe the question that users need to answer. This will make it difficult for screen-reader users to understand the page.
+  - Welsh users will have difficulties when using the Welsh version, as ‘skip to main content’ copy is not translated into Welsh.
+  - On the ‘Evidence to support this appeal (optional)’ page, screen-reader users are not aware about any changes on the page when they upload a file or activate the ‘add another file’ button.
+milestones:
+  - description: ‘file 1’ text is marked up as a label, when associated input is missing. This fails WCAG 2.1 success criterion 4.1.2 Name, role, value
+    date: 2023-05-13
```
Comment (LewisMitchell) [inline conf/services/penalties-appeals]: This 'date' field needs to be indented again

File: conf/services/penalties-appeals [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - description: ‘This penalty was issued more than 30 days ago’ copy is marked up as a label. It does not describe the purpose of the label. This fails WCAG 2.1 success criterion 2.4.6 Headings and labels
+      date: 2023-05-13
+  - description: when viewing the page in Welsh some parts of the service are missing the necessary translations. This fails WCAG 2.1 success criterion 3.1.2 Language of parts
+    date: 2023-05-13
+  - description: file upload process missing live region implementation of status messages, that would help screen-reader users to be notified about change and upload progress. This fails WCAG 2.1 success criterion 4.1.3 Status messages
+    date: 2023-05-13
```
Comment (LewisMitchell) [inline conf/services/penalties-appeals]: image
Some minor grammatical changes needed - see screenshot

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/services/penalties-appeals (hunk 1 above)
Comment (JBlacklaws) [reply conf/services/penalties-appeals]: Fixed, along with date positioning

File: conf/services/penalties-appeals.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+serviceName: VAT Penalties and Appeals Reform
+serviceDescription: VAT-registered businesses, and their agents, can see penalties for late submission of VAT Returns and late payment of VAT, and appeal against those penalties online.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /penalties-appeals.yml
```
Comment (LewisMitchell) [inline conf/services/penalties-appeals.yml:4]: image
This should be just `/penalties-appeals`

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**